src/App.tsx

```diff
@@ -96,16 +96,7 @@ const App: React.FC = () => {
             data={[
               {
                 name: 'Home',
-                subMenu: [
-                  {
-                    name: 'Lorem Home',
-                    link: '#'
-                  },
-                  {
-                    name: 'Morem',
-                    link: '#'
-                  }
-                ]
+                link: '/'
```

Comment:
I'd suggest to rename this property to `url`. Link is the element that navigates to the given URL.